### PR TITLE
SPCTR-639 - Fix: Inline Notice - R3 Issues

### PR DESCRIPTION
**Pull request by bsf-zanev**
### Description
Debugged and Fixed P1 Issues reported during Acceptance Testing R:3 for the Inline Notice Block.

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
1. Ran all tests required on the Editor and Front-end as per the issues.
2. Tested Left and Right Alignment for Modern and Classic Layouts on all 3 Device Types with varied Title Padding and Border Widths.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

**Comment by RajkiranBagal** [comment]
@supriyajjadhav Can you check Copy Paste attributes in this PR? Seems wrong to me.

File: src/blocks/inline-notice/attributes.js [hunk 1]
```diff
@@ -74,6 +74,32 @@ const attributes = {
 		type: 'string',
 		default: 'fa fa-window-close',
 	},
+	iconSize: {
+		type: 'number',
+		default: 16,
+		UAGCopyPaste: {
+			styleType: 'btn-width'
```
Comment (supriyajjadhav) [inline src/blocks/inline-notice/attributes.js:81]: We have used 'btn-size' globally. Please make the changes accordingly.

Comment (bsf-zanev) [reply src/blocks/inline-notice/attributes.js:81]: @supriyajjadhav I could not find 'btn-size' in the Copy/Paste Doc.

For responsive, I'll use 'btn-size-tablet' and 'btn-size-mobile' respectively.

Comment (bsf-zanev) [reply src/blocks/inline-notice/attributes.js]: Updated.

**Review by RajkiranBagal — APPROVED** [review]

**Review by supriyajjadhav — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**